templates/modals/edit_user.html

```diff
@@ -72,13 +60,13 @@
 
     <div class="form-group" ng-class="{'has-error': errors.password, 'required': !editUserModel.id}">
       <label for="password" translate>Password</label>
-      <input id="password" type="password" class="form-control" ng-model="editUserModel.password" autocomplete="off" ng-disabled="editUserModel.type === '_admin'"/>
```

Comment:
I think we still need this.

This is so you can't attempt to change the password of a CouchDB admin, which you can't do anyway because they aren't primarily stored in the `_users` DB.

Reply:
Yeah, you're right - added this back.